app/models/notifications/queries/AssociatedConsignmentsInfo.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.notifications.queries
+
+case class AssociatedConsignmentsInfo(childDucrs: Seq[DucrInfo] = Seq.empty, childMucrs: Seq[MucrInfo] = Seq.empty) {
+
+  def size: Int = childDucrs.size + childMucrs.size
+
+  def mostSevereRoe: Option[String] =
```

Comment:
This is a temp. solution to extract a ROE - it's not the most severe and might not be needed at all if we show all of the children as a list